src/catalogue/oci/build.gradle

```diff
@@ -34,7 +44,9 @@ dependencies {
     // Database
     implementation("io.micronaut.data:micronaut-data-jdbc")
     implementation("io.micronaut.oraclecloud:micronaut-oraclecloud-atp")
-    implementation platform("com.oracle.database.jdbc:ojdbc-bom:21.1.0.0")
+    implementation platform("com.oracle.database.jdbc:ojdbc-bom:23.3.0.23.09")
```

Comment:
I don't think you need to set an explicit version, it should come from the Micronaut BOM.

Can you please try without a version?

Reply:
Actually when I remove the version  the build fails  
<img width="414" alt="Screenshot 2024-03-14 160846" src="https://github.com/oracle-quickstart/oci-micronaut/assets/161726330/4abb82e9-0e3f-4c76-8288-c878412f10dd">